src/main/config/messages_template.xml

```diff
@@ -438,409 +440,409 @@ See the accompanying LICENSE file for applicable license.
     <reason>Conflict text style is applied on the current element based on DITAVAL flagging rules.</reason>
-    <response>Please check ditaval and dita source to make sure there is no style conflict on the element which needs to be flagged.</response>
+    <response>Check the DITAVAL and DITA source files to make sure there is no style conflict on the element that needs to be flagged.</response>
   </message>
-  
+
   <message id="DOTX055W" type="WARN">
-    <reason>Customized stylesheet uses deprecated template "flagit". Conditional processing is no longer supported using this template.</reason>
-    <response>Please update your stylesheet to use template "start-flagit" instead of deprecated template "flagit".</response>
+    <reason>A customized stylesheet uses the deprecated 'flagit' template. Conditional processing is no longer supported using this template.</reason>
+    <response>Update the stylesheet to use the 'start-flagit' template instead of the 'flagit' template.</response>
   </message>
-  
+
   <message id="DOTX056W" type="WARN">
-    <reason>The file '%1' is not available to resolve link information.</reason>
+    <reason>The '%1' resource is not available to resolve link information.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX057W" type="WARN">
-    <reason>The link or cross reference target '%1' cannot be found, which may cause errors creating links or cross references in your output file.</reason>
+    <reason>The link or cross reference target '%1' cannot be found, which may cause errors in the output.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTX058W" type="WARN">
-    <reason>No glossary entry was found associated with key '%1' on %2 element. The build will try to determine the best display text and hover text for terms and abbreviations.</reason>
-    <response></response>
+    <reason>No glossary entry found for the '%1' key on the '%2' element.</reason>
+    <response>Check display text and hover text for terms and abbreviations.</response>
   </message>
-  
+
   <message id="DOTX060W" type="WARN">
-    <reason>Key '%1' was used in an abbreviated-form element, but the key is not associated with a glossary entry.</reason>
-    <response>Abbreviated-form should ONLY be used to reference to a glossary entry.</response>
+    <reason>The '%1' key is used in an &lt;abbreviated-form&gt; element, but the key is not associated with a glossary entry.</reason>
+    <response>Abbreviated-form should ONLY be used to reference a glossary entry.</response>
   </message>
-  
+
   <message id="DOTX061W" type="WARN">
-    <reason>ID '%1' was used in topicref tag but did not reference a topic element. The href attribute on a topicref element should only reference topic level elements.</reason>
-    <response></response>
+    <reason>The @id attribute value '%1' is defined on a &lt;topicref&gt; element, but it does not reference a topic element.</reason>
```

Comment:
This is not quite right ... issue is with something like 
`<topicref href="file.dita#section"/>`
where `section` is not a topic ID, it's some other element. The updated message implies the problem is with the `@id` attribute on the topicref, but instead it's with the ID portion of the `@href` reference